assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Footer.js

```diff
@@ -0,0 +1,170 @@
+	MODULES_ANALYTICS,
+	DATE_RANGE_OFFSET as DATE_RANGE_OFFSET_ANALYTICS,
+} from '../../../../analytics/datastore/constants';
+import { generateDateRangeArgs as generateAnalyticsDateRangeArgs } from '../../../../analytics/util/report-date-range-args';
+import SourceLink from '../../../../../components/SourceLink';
+import Data from 'googlesitekit-data';
+const { useSelect } = Data;
+
+const Footer = ( { metrics, selectedStats } ) => {
+	const propertyID = useSelect( ( select ) =>
+		select( MODULES_SEARCH_CONSOLE ).getPropertyID()
+	);
+	const isDomainProperty = useSelect( ( select ) =>
+		select( MODULES_SEARCH_CONSOLE ).isDomainProperty()
+	);
+	const referenceSiteURL = useSelect( ( select ) =>
+		untrailingslashit( select( CORE_SITE ).getReferenceSiteURL() )
+	);
+	const url = useSelect( ( select ) =>
+		select( CORE_SITE ).getCurrentEntityURL()
+	);
+	const dateRangeDates = useSelect( ( select ) =>
+		select( CORE_USER ).getDateRangeDates( {
+			offsetDays: DATE_RANGE_OFFSET,
+		} )
+	);
+
+	const searchConsoleDeepLinkArgs = {
+		resource_id: propertyID,
+		metrics: metrics[ selectedStats ]?.metric,
+		...generateDateRangeArgs( dateRangeDates ),
+	};
+	if ( url ) {
+		searchConsoleDeepLinkArgs.page = `!${ url }`;
+	} else if ( isDomainProperty && referenceSiteURL ) {
+		searchConsoleDeepLinkArgs.page = `*${ referenceSiteURL }`;
+	}
+	const searchConsoleDeepLink = useSelect( ( select ) =>
+		select( MODULES_SEARCH_CONSOLE ).getServiceReportURL(
+			searchConsoleDeepLinkArgs
+		)
+	);
+
+	const analyticsDrilldownArgs = [];
+	if ( isURL( url ) ) {
+		analyticsDrilldownArgs.push(
+			`analytics.pagePath:${ getURLPath( url ) }`
+		);
+	}
+	const analyticsRangeDates = useSelect( ( select ) =>
+		select( CORE_USER ).getDateRangeDates( {
+			compare: true,
+			offsetDays: DATE_RANGE_OFFSET_ANALYTICS,
+		} )
+	);
+	const analyticsGoalsDeepLink = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getServiceReportURL(
+			'conversions-goals-overview',
+			{
+				...generateAnalyticsDateRangeArgs( analyticsRangeDates ),
+				'_r.drilldown': analyticsDrilldownArgs.join( ',' ),
+			}
+		)
+	);
+
+	const analyticsVisitorsOverviewDeepLink = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getServiceReportURL( 'visitors-overview', {
+			...generateAnalyticsDateRangeArgs( analyticsRangeDates ),
+			'_r.drilldown': analyticsDrilldownArgs.join( ',' ),
+		} )
+	);
+
+	const analyticsVisitorsDeepLink = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getServiceReportURL(
+			'acquisition-channels',
+			{
+				...generateAnalyticsDateRangeArgs( analyticsRangeDates ),
+				'_r.drilldown': [
+					...analyticsDrilldownArgs,
+					'analytics.trafficChannel:Organic Search',
+				].join( ',' ),
+			}
+		)
+	);
+
+	const analyticsDeepLinks = {
+		users: analyticsVisitorsDeepLink,
+		goals: analyticsGoalsDeepLink,
+		'bounce-rate': analyticsVisitorsOverviewDeepLink,
+	};
+
+	const { service, id } = metrics?.[ selectedStats ];
+
+	return (
+		<Fragment>
```

Comment:
Maybe we don't need this `<Fragment>` element? Just return do it something like this:

```js
if ( service === 'search-console' ) {
    return (
        <SourceLink ... />
    );
}

return (
    <SourceLink href={ analyticsDeepLinks[ id ] } ... />
);
```

Reply:
Sounds good - that's done too. Cheers